### DISCUSSION TITLE
How to use the Relationships Tab

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#594] [HOW TO...]
**DEPRECATED** [Relationships Data & Workflows](https://docs.google.com/document/d/190msXiIYgui4zReB4O8-IQXq4Gue0GqQZU3wMCZG_jo/)
- Formerly known as “Stomach Contents (both directions) data mapping/key”

**DEPRECATED** [Relationship Data: Pre-Dev Workflow in EMu](https://docs.google.com/document/d/1zvmyEmAilPAmcY1MF-m1I9ZVlDqL6ah170nUez-kR4k)

**Comment by fmjjones** [comment]
**FINAL** [Workflow for Relationship Tab RelNHTab](https://docs.google.com/document/d/190msXiIYgui4zReB4O8-IQXq4Gue0GqQZU3wMCZG_jo/edit?usp=sharing)
